src/mmw/js/src/modeling/models.js

```diff
@@ -888,6 +889,7 @@ var ScenarioModel = Backbone.Model.extend({
 var ScenariosCollection = Backbone.Collection.extend({
     model: ScenarioModel,
     comparator: 'created_at',
+    alerts: [],
```

Comment:
I recall there was some odd issue I encountered when I initially tried using a single object, but in any event this just seemed like a slightly cleaner approach. Since it's attached to a collection type, a "mailbox" seemed more idiomatic. More tangibly, I didn't like the the idea of touching the state in the view that this binds to, e.g. clearing the value if it were just a string, so using list constructs seemed preferable.